src/partials/icons/menu.svg

```diff
@@ -1 +1 @@
-<svg viewBox="0 0 24 24"><title>Menu</title><path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"/></svg>
+<svg viewBox="0 0 24 24" aria-hidden="true"><path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"/></svg>
```

Comment:
from all the changes in this PR, this is possibly the only one i'd keep (and for the rest consider looking more at how the bootstrap offcanvas component now operates)